static/app/views/settings/projectPerformance/projectPerformance.tsx

```diff
@@ -136,10 +137,12 @@ class ProjectPerformance extends AsyncView<Props, State> {
   renderBody() {
     const {organization, project} = this.props;
     const endpoint = `/projects/${organization.slug}/${project.slug}/transaction-threshold/configure/`;
+    const editAccessLevel: Scope[] = ['member:write'];
```

Comment:
🤔 why `'member:write'`? If performance settings are per project this feels like it wants to be `'project:write'`, at least that's what other panels seem to be doing

Also, small nit, but I think a name like `requiredScopes` or something similar would better explain the variable's significance

Reply:
Nice catch! I actually did mean to write project:write (to be have the same level as the permissionAlert)

I chose to use `editAccessLevel` as in the access level required to edit the form and to be consistent with the prop name.
Access is more consistent with the prop name, scope is more consistent with type. 
But I'll go with your suggestion, I like it

I'd be more inclined to change the prop name for the Acess component to `scope`.
 A component named Access, with a prop called access is not as specific/clear as a component named Access with a prop called scope.
 
 All little nitpicks tho